src/index.js

```diff
@@ -1,50 +1,65 @@
-import Autocomplete from './components/Autocomplete/Autocomplete';
+import { buildStatefulComponent } from './util/state-management';
```

Comment:
I'm okay with it. We should probably have a common unit test that enforces the convention though.